examples/seismic/viscoacoustic/viscoacoustic_example.py

```diff
@@ -41,6 +43,14 @@ def test_viscoacoustic():
     assert np.isclose(norm(rec), 18.7749, atol=1e-3, rtol=0)
 
 
+@pytest.mark.parametrize('ndim', [1, 2, 3])
+def test_viscoacoustic_stability(ndim):
```

Comment:
add kernel as parameter to test all three

Reply:
is this the only thing missing here? @mloubout @georgebisbas

Reply:
I fixed the other ones so if wanted yes, can always be added later if wanted and just merge

Reply:
still missing?

Reply:
Yes it is still missing.